promises.cf

```diff
@@ -57,6 +57,10 @@ body common control
 
       version => "CFEngine Promises.cf 3.7.0";
 
+      cfengine_3_7.(debian||redhat)::
```

Comment:
Additionally, the CFEngine version selection should be exclusive rather than inclusive here as well, just like in `` inventory/any.cf``.